server/minify.mjs

```diff
@@ -1,4 +1,5 @@
 import fs from 'fs';
```

Comment:
This whole file got pretty messed up by your editor. Like UltimateGeek said, please make sure to keep indentation at two spaces.

Also: the only change in this file is to replace /tmp by os.tmpdir. That fix is already commited to main. You should - I think - rebase this pull request to the latest revision of main.